src/FlatFile.FixedLength/Implementation/FixedLengthFileMultiEngine.cs

```diff
@@ -138,6 +138,27 @@ protected override bool TryParseLine<TEntity>(string line, int lineNumber, ref T
         public void Read(Stream stream)
         {
             var reader = new StreamReader(stream);
+            ReadInternal(reader);
+        }
+                
+        /// <summary>
+        /// Reads the specified stream reader.
+        /// </summary>
+        /// <param name="streamReader">The stream reader.</param>
+        /// <exception cref="ParseLineException">Impossible to parse line</exception>
+        public void Read(StreamReader reader)
```

Comment:
I'm not sure if this should be part of the contacts. We already have the ability to pass in a stream.

Reply:
I actually would like the feature as well. I needed to use a custom StreamReader implementation for reading my files and ended up downcasting the engine to use that overload.